src/features/homePage/Filter/Sort/Sort.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { sortMovies } from '../../actions';
+
+import styles from './Sort.module.scss';
+
+const setIconClass = (currentSortType, direction, sortType) => {
+  if (currentSortType === sortType) {
+    return direction ? 'fa fa-chevron-up' : 'fa fa-chevron-down'
+  }
+
+  return 'fa fa-minus';
+}
+
+const Sort = (props) => {
+  const {
+    onSortMovies,
+    sortType,
+    sortedByAscend
+  } = props;
+
+  const sortLikesIconClass = setIconClass(sortType, sortedByAscend, 'likes');
+  const sortStarsIconClass = setIconClass(sortType, sortedByAscend, 'stars');
+
+  return (
+    <div className={styles.filter}>
+      <button
+        className={styles.sortButton}
+        onClick={() => onSortMovies('likes')}>
```

Comment:
As a tip and option. Very oftern such things, I mean argument `likes` moves to constants. It can be constant inside a component file, or in a separate file with constants. For ex:
const LIKES = 'likes';
...onClick={() => onSortMovies(LIKES)}